.changeset/slimy-cooks-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nordeck/matrix-neoboard-react-sdk': patch
+---
+
+Fixes placing a rounded rectangle with a click
```

Comment:
Does it make sense to add a changeset for a fix where the error was never published?

Reply:
yea, guess not. will remove it.